spec/factories/vacancies.rb

```diff
@@ -121,13 +123,5 @@
       expires_on { Faker::Time.between(from: Time.zone.today + 10.days, to: Time.zone.today + 20.days) }
       expiry_time { nil }
     end
-
-    trait :first_supporting_subject do
```

Comment:
Maybe we ought to keep these attributes in the factory for now, if we've still got these attributes on some records?

Reply:
Vacancies won't be created with these attributes anymore, so I don't think it makes sense to add them by default in the factory